docs/assets/bindings-delete-flow.drawio

```diff
@@ -0,0 +1,101 @@
+<mxfile host="Electron" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/24.7.17 Chrome/128.0.6613.36 Electron/32.0.1 Safari/537.36" version="24.7.17">
```

Comment:
Term `DB binding` seems cryptic.
Binding in DB, binding record from DB, in DB.
I am not sure.

Reply:
or "existing binding"?

Reply:
The thing is that we have different bindings. There is binding and its representations in database.

Its ambiguous.

Reply:
`DB Binding exists` = `Binding already stored in DB`?